src/clr-core/icon-shapes/icon.element.scss

```diff
@@ -315,24 +314,31 @@ svg {
 // DEPRECATED IN 3.0: has-badge--danger, has-badge--error
 :host(.has-badge--danger),
 :host(.has-badge--error),
-:host([badge-type='danger']) {
+:host([badge='danger']) {
   .clr-i-badge {
     @include setIconColor(#{'var(--clr-icon-color-danger, var(--clr-color-danger-700,' + $clr-color-danger-700 + '))'});
   }
 }
 
 // there is no .has-badge--warning classname
-:host([badge-type='warning']) {
+:host([badge='warning']) {
   .clr-i-badge {
     @include setIconColor(
       #{'var(--clr-icon-color-warning, var(--clr-color-warning-900,' + $clr-color-warning-900 + '))'}
     );
   }
 }
 
+:host([badge='inherit']) {
+  .clr-i-badge {
+    @include setIconColor('inherit');
+  }
+}
+
 // DEPRECATED IN 3.0: has-badge--info
 :host(.has-badge--info),
-:host([badge-type='info']) {
+:host([badge='info']),
+:host([badge='highlight']) {
```

Comment:
Is highlight the same as info? If so should we encourage teams to migrate to using info and not carry highlight over from the previous clr-icon API?

Reply:
We can. `highlight` is a holdover that isn't widely used. Even `info` is used only sparingly. `highlight` less so.

Reply:
done!